demo/meeting/serverless/deploy.js

```diff
@@ -114,14 +114,16 @@ function appHtml(appName) {
 
 function ensureApp(appName = app) {
   console.log(`Building ${appName} application`);
-  spawnOrFail('npm', ['run', 'build'], { cwd: path.join(__dirname, '..') });
-  spawnOrFail('npm', ['run', 'build:prod'], { cwd: path.join(__dirname, '..') });
+  const output = spawnOrFail('npm', ['run', 'build'], { cwd: path.join(__dirname, '..') });
+  console.log(output);
```

Comment:
Can we pass in a flag to suppress this instead?

Reply:
Done